README.md

```diff
@@ -7,8 +7,26 @@
 ## Quick start
 
 ## Documentation
+### - Available features in Amino:
+- [Style generation (generate style constants and css file)](./build-utils/css/README.md)
+- [Icon component generation process (generate optimized react components based on SVG downloaded from Figma/any sources)](./svgReact/icons/README.md)
+- [Flag component generation process (generate optimized react components for flag svgs from smallest svg size of AWS and Figma)](./svgReact/flags/README.md)
```

Comment:
Thank you for updating the Readme 💯

Reply:
I added a story for Date input already. Thanks for catching that!